pkgs/applications/networking/cluster/terraforming/Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
+
+gem 'terraforming'
```

Comment:
Ideally we would pin this gem to make the lockfile more reproducible.

Reply:
I don't think that's necessary, in fact, I think it'll do more harm than good. The Gemfile.lock/gemset.nix already lock to a specific version and whenever I run the updater script I expect to just grab the latest and greatest for me to test and submit as PR; Having to explicitly have to edit the Gemfile defeats the whole point of the updaterScript. The only useful usecase I can see for this is to roll out a bug/security fix for a specific pinned version which we can do temporarily at the time of that update.